pkgs/build-support/fetchzip/default.nix

```diff
@@ -5,7 +5,10 @@
 # (e.g. due to minor changes in the compression algorithm, or changes
 # in timestamps).
 
-{ lib, fetchurl, unzip }:
+# Have to pull in glibcLocales for unzip in setup-hook.sh to handle
+# UTF-8 aware locale:
```

Comment:
Please move the comment to nativeBuildInputs.

Reply:
Sounds good! Moved.